src/index.js

```diff
@@ -1,24 +1,23 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import useCPU from './hooks/use-cpu';
 import RegisterPanel from './components/Registers';
+// import useCPU from './hooks/use-cpu';
```

Comment:
can we remove code instead of commenting out, unless its inconvenient not to?

Reply:
Sure, just left it for quick reference if I need to include it